doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -178,14 +183,18 @@ def plot_samples(samples, y_axis_label):
     plt.ylabel(y_axis_label)
 
 
-plot_samples(samples, "unconstrained_variables_values")
-plot_samples(parameter_samples, "parameter_values")
+plot_samples(samples, model, "unconstrained_variables_values")
+plot_samples(parameter_samples, model, "parameter_values")
 
 
 # %% [markdown]
 # You can also inspect the marginal distribution of samples.
 
 # %%
+
+param_to_name = {param: name for name, param in gpflow.utilities.parameter_dict(model).items()}
```

Comment:
how about moving this into `marginal_samples` below as you did for the previous function (change above)?

Reply:
yes, can do